### DISCUSSION TITLE
vAGS-4.1.1-AU-1.2.0

### DISCUSSION THREAD

**Discussion by SolidRockProgrammer** [australiangeomechanics/agsformat#46] [Announcements]
Officially referred to as AGS 4.1.1 AU 1.2
Release date: 16/06/2022

This replaces AGS 4.1.0 AU 1.1.2. 

<hr /><em>This discussion was created from the release <a href='https://github.com/australiangeomechanics/agsformat/releases/tag/'>vAGS-4.1.1-AU-1.2.0</a>.</em>

**Comment by saadbaraah5-stack** [comment]
can you help me and provide the gINT files for (AGS 4.1.1)

**Reply by SolidRockProgrammer** [reply]
Hi
The Datgel DGD Tool Pro gINT Add-In is the only off the shelf option with gINT to support AGS 4.1.1 AU format
https://www.datgel.com/datgel-dgd-tool
regards
Phil Wade

**Comment by saadbaraah5-stack** [comment]
Thank you
But can you tell me how to download it from the website ?

**Reply by SolidRockProgrammer** [reply]
You can email sales@datgel.com to enquire about Datgel software
This page explains how to trial the Datgel DGD Tool: https://docs.datgel.com/dgd-tool/latest/professional/quick-start-guide

**Comment by RobDayArup** [comment]
Hi - could you please advise if there are any plans to update AGS4.1.1 AU1.2 for compatibility with AGS4.2 released December 2025?  AGS4.1.1 has now been deprecated and organizations are adopting AGS4.2in the UK.  Just this week Bentley have released an AUS_03.01 Config pack for OpenGround Australia users incorporating the AGS4.2 changes.  I'm concerned adopting this will break compatibility with the Australian 4.1.1 schema, particularly with the deprecated static cone groups (SCPx replaced by CPTx).
